service/controllers/userController.js

```diff
@@ -101,4 +105,49 @@ const current = async (req, res, next) => {
   });
 };
 
-module.exports = { signUp, logIn, logOut, current };
+const uploadAvatar = async (req, res, next) => {
+  const { path: avatarDir } = req.file;
+
+  await Jimp.read(avatarDir)
+    .then((avatar) => {
+      return avatar.cover(250, 250).write(avatarDir);
+    })
+    .catch((err) => {
+      next(err);
+    });
```

Comment:
Może użyj jednego sposbu? czyli try catch z await ale bez then żeby gdyby był błąd to go przechwycić.